.github/workflows/build.yml

```diff
@@ -59,18 +59,19 @@ jobs:
           cp -r /Volumes/Eclipse/Eclipse.app /Applications/
           hdiutil detach /Volumes/Eclipse
           echo "eclipseRoot.dir=/Applications/Eclipse.app" | tee eclipsePlugin/local.properties
-      - name: Pre-build setup
-        if: matrix.os != 'windows-latest'
-        run: |
-          echo "keystorepass=${KEYSTORE_PASS}" >> gradle.properties
-          git fetch --no-tags https://$GITHUB_TOKEN@github.com/spotbugs/spotbugs.git +refs/heads/master:refs/remotes/origin/master
-          if [ "$GPG_SECRET_PASSPHRASE" != "" ]; then
-            gpg --quiet --batch --yes --decrypt --passphrase="$GPG_SECRET_PASSPHRASE" --output spotbugs.jks .github/workflows/spotbugs.jks.gpg;
-          fi
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GPG_SECRET_PASSPHRASE: ${{ secrets.GPG_SECRET_PASSPHRASE }}
-          KEYSTORE_PASS: ${{ secrets.KEYSTORE_PASS }}
+      # Disable gpg signing on *nix for time being until we get new keys established, checks will be on windows only for now
```

Comment:
This disables the decryption of the keystore (which contains the expired letsencrypt certificate used to sign the Eclipse plugin), it does not disable the gpg signing, right?